hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/fs/shell/Tail.java

```diff
@@ -107,16 +109,15 @@ private long dumpFromOffset(PathData item, long offset) throws IOException {
     if (offset < 0) {
       offset = Math.max(fileSize + offset, 0);
     }
-    
-    FSDataInputStream in = item.fs.open(item.path);
-    try {
+    // Always do sequential reads.
```

Comment:
The comment says do sequential reads, but then the sequential option isn't passed below.